nerdlets/create-flow/blank.js

```diff
@@ -1,45 +1,23 @@
 import React, { useEffect, useMemo, useState } from 'react';
 import PropTypes from 'prop-types';
 
-import { Button, HeadingText, TextField } from 'nr1';
+import { Button, HeadingText, InlineMessage, TextField } from 'nr1';
 
 import { Select } from '../../src/components';
 import { REFRESH_INTERVALS } from '../../src/constants';
 
-const BlankFlow = ({ accountId, accounts = [], onCreate, onCancel }) => {
+const BlankFlow = ({ accountId, accountName, onCreate, onCancel }) => {
   const [flowName, setFlowName] = useState('');
   const [selectedAccountId, setSelectedAccountId] = useState();
+  const [selectedAccountName, setSelectedAccountName] = useState();
```

Comment:
Use the `accountName` prop instead of this state variable. This was only needed when we handled changing the account from within the component.